distribute.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -euo pipefail
+IFS=$'\n\t'
+mkdir dist
+echo "Building Way Cooler..."
```

Comment:
This script should not be ran by users (except for those that want to emulate/test our build system for distribution). This should only be ran by us, I was planning for it to be integrated into the build system kicked off by `ci.py`, however I didn't want to go through the trouble of setting up travis to build this all correctly (it took me long enough to get working on my local machine...). This is something I will correct in the future, when the pain of building the releases manually on my machine becomes too much.

As for your first bullet point, again here's the general layout of how I envisioned the distribution system:

0) `distribute.sh`, ran only by us
1) `curl https://way-cooler.org/install-script.sh | sh`, not built yet and _optional_. This script will be downloaded to the user's machine, it will do another curl for the most up-to-date version packaged by `distribute.sh`, and then run the second stage.
2) `install.sh`, ran by users, either manually or through stage 1.